src/serializers.ts

```diff
@@ -4,7 +4,7 @@ import express from 'express'
 
 export const serializers: bunyan.StdSerializers = {
 
-  event: (event: Webhooks.WebhookEvent<any> | any) => {
+  event: (event: Webhooks.WebhookEvent<any>) => {
```

Comment:
This one actually needs to stay. The `Webhooks.WebhookEvent<any>` is redundant because of `any`